rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -420,6 +421,8 @@ static void check_opts(const char *opt)
 		no_sgx_flc = true;
 	else if (!strcmp(opt, "debug"))
 		debugging = true;
+	else if (!strcmp(opt, "sgxmmap"))
```

Comment:
This parameter is not necessary right? If both attr->fd and attr->addr are valid, using enclave share mapping is already implied.

Reply:
ok, thanks